integration/helpers.go

```diff
@@ -1228,30 +1214,30 @@ func (i *TeleInstance) NewUnauthenticatedClient(cfg ClientConfig) (tc *client.Te
 }
 
 // NewClient returns a fully configured and pre-authenticated client
-// (pre-authenticated with server CAs and signed session key)
+// (pre-authenticated with server CAs and signed session key).
 func (i *TeleInstance) NewClient(t *testing.T, cfg ClientConfig) (*client.TeleportClient, error) {
```

Comment:
I find it odd for a func to accept a `testing.T` **and** return an error.

If there's an error, shouldn't it just fail the test?

Reply:
Just saw that, updated to no longer accept `testing.T` and instead return an error and have the caller make the decision.